pom.xml

```diff
@@ -197,13 +205,15 @@
                 <version>3.1.1</version>
                 <configuration>
                     <!-- During release:perform, enable the "release" profile (see below) -->
-                    <releaseProfiles>release</releaseProfiles>
+                    <releaseProfiles>-Drelease</releaseProfiles>
```

Comment:
@nwoodward : I think this might need to be specified as `<arguments>-Drelease</arguments>`  That would match the main POM: https://github.com/DSpace/DSpace/blob/main/pom.xml#L425